routes/api.js

```diff
@@ -1,6 +1,7 @@
 // Dependencies
```

Comment:
My preference would be to have the yelp endpoints in their own module (e.g. routes/yelp.js) and have use import them in server how we do api and html (require("./routes/yelp)(app)). I'll let you determine if that's how we want to organize the code.